docs/Migrating-v3.md

```diff
@@ -116,6 +116,20 @@ Example: `@ecl-twig/ec-component-accordion/ecl-accordion.html.twig` => `@ecl/acc
 
 [ECL Compliance component](https://github.com/ec-europa/ecl-twig/tree/master/src/ec/packages/ec-component-ecl-compliance) has been removed.
 
+## Presets / themes
+
+A new concept of "theme" has been introduced, which complete the presets used in ECL 2. Presets are still used under the hook (one for each theme), but what is exposed to end users is now the following:
+
+- ec-core: contains everything related to EC sites
+- eu-core: contains everything related to EU sites
+
+Some of the old v2 presets have been removed or modified:
+| Modified presets | Status |
```

Comment:
```suggestion
| Preset | Status |
```

As it's already mentioned that they have been modified, the important info is how